### ISSUE TITLE
This crate does not work with the new workspace dependencies 

### ISSUE THREAD

**Issue by Jasperav** [est31/cargo-udeps#140]
The new workspace deps feature is explained here: https://betterprogramming.pub/workspace-inheritance-in-rust-65d0bb8f9424

Reproduction path:
1. Add random dependencies to your root Cargo.toml
2. Run this library
3. 'All deps seem to be used' is shown, although the deps are not used.

**Comment by domenicquirl** [comment]
When using 
```toml
[package]
version.workspace = true
```
in a workspace crate, I'm getting 
```
error: failed to load manifest for workspace member `<path>`

Caused by:
  failed to parse manifest at `<path>/Cargo.toml`

Caused by:
  invalid type: map, expected SemVer version for key `package.version`
```

with both a nightly from 1.64 (`rustc 1.64.0-nightly (7425fb293 2022-06-30)`) and current (`rustc 1.66.0-nightly (ce7f0f1aa 2022-09-28)`).

**Comment by larry0x** [comment]
I can confirm latest version works with workspace deps

**Comment by woshilapin** [comment]
I also tested that it works fine on version `0.1.34`. However, the release of the binaries never happened. The tag is present [here](https://github.com/est31/cargo-udeps/tree/v0.1.34), the crate is published [here](https://crates.io/crates/cargo-udeps/0.1.34), but there is no release [here](https://github.com/est31/cargo-udeps/releases).

It seems like the Github Actions on the job somewhat failed (see [here](https://github.com/est31/cargo-udeps/actions/runs/3109006480/jobs/5038809722)).

**Comment by est31** [comment]
@woshilapin yeah I have botched that update a little, unfortunately, forgot to check in the lockfile changes. As there is a Rust release next week however, there would also be a new cargo release, which means that it's also time for a new cargo-udeps release. The train model definitely has its advantages :).

**Comment by est31** [comment]
0.1.35 is out: https://github.com/est31/cargo-udeps/releases/tag/v0.1.35

**Comment by Jasperav** [comment]
@est31 It does work when you reference a lib from the root Cargo.toml in another project in your workspace and it is not used. However, libs that are inside the root Cargo.toml file but nowhere referenced in other projects in the workspace are not picked up. Can this be fixed?

**Comment by est31** [comment]
@Jasperav that's interesting, thanks for the feedback. I guess it should be possible to support that, it's more of a feature request than a bug however. The reason why I'm keeping *this* issue open is because there is no test yet for unused workspace dependencies that cargo builds.

**Comment by bushrat011899** [comment]
Is there any movement on support for detecting unused workspace dependencies? We'd like to consider their adoption in Bevy but the lack of detection for unused items is a total blocker.
